pmlc/target/x86/CMakeLists.txt

```diff
@@ -0,0 +1,9 @@
+pml_mlir_transforms(
+  SRCS
+    "heatmap_impl.cc"
```

Comment:
Why the quotes?

Reply:
This was an IREE convention, but as I'm refactoring, I'll remove the quotes (they don't seem to be necessary)